AUTHORS.md

```diff
@@ -3,7 +3,7 @@
 ## Development Lead
 
 - Max Resnick [maxres-fr](https://github.com/maxres-fr)
-- Lee Baines  [lee-bains](https://github.com/lee-baines)
+- Lee Baines  [lee-baines](https://github.com/lee-baines)
 - Jhonattan Castillo [jrcast](https://github.com/jrcast)
 - Warren Strange [wstrange](https://google.com/wstrang)
```

Comment:
well, I have zero contributions to the cli, but heck I'll take some credit

Reply:
Maybe https://github.com/wstrange